fbw-a32nx/src/behavior/src/A32NX_Interior_RMP.xml

```diff
@@ -144,26 +141,7 @@
         <Component ID="PUSH_RADIO#SIDE#_SEL_SEQ1" Node="PUSH_RADIO#SIDE#_SEL_SEQ1">
             <UseTemplate Name="ASOBO_GT_Emissive_Gauge">
                  <POTENTIOMETER>7</POTENTIOMETER>
-                <EMISSIVE_CODE type="rnp" return="number">
-                    let indicatorsPowered = #INDICATORS_POWERED, bool#;
-                    let id = #ID, number#;
-                    alias toggleSwitch = (L:A32NX_RMP_#SIDE#_TOGGLE_SWITCH, bool);
-                    alias selectedModeLeft = (L:A32NX_RMP_L_SELECTED_MODE, number);
-                    alias selectedModeRight = (L:A32NX_RMP_R_SELECTED_MODE, number);
-                    alias annunciatorLight = (L:A32NX_OVHD_INTLT_ANN, number);
-                    alias dc2Powered = (L:A32NX_ELEC_DC_2_BUS_IS_POWERED, bool);
-                    let emissiveDim = if annunciatorLight == 2 { 0.05 } else { 1 };
-                    // @todo abnormal defined differently with NAV modes available.
-
-                    let rmpLeftAbnormal = id == 1 and (selectedModeLeft == 2 or selectedModeLeft == 3 or selectedModeRight == 1);
-                    let rmpRightAbnormal = id == 2 and (selectedModeRight == 1 or selectedModeRight == 3 or selectedModeLeft == 2);
-
-                    (if (toggleSwitch and (rmpLeftAbnormal or rmpRightAbnormal) or ((annunciatorLight == 0) and dc2Powered)) and indicatorsPowered {
-                        1 * emissiveDim
-                    } else {
-                        0
-                    })
-                </EMISSIVE_CODE>
+                <EMISSIVE_CODE>(L:A32NX_RMP_SEL_LIGHT_ON)</EMISSIVE_CODE>
```

Comment:
This looks very problematic. You have deleted all the electrical logic here. You have made a single Lvar written by both RMPs which will not work properly when only one side is powered. See the further problem below as well...

Reply:
Indeed. Very stupid. 

The problem is even more complex than that because if the sel light is on and we swtich off the onside RMP, the sel light remains on on the other one. I'll code that in Rust